examples/docker-without-nfs/docker-compose.yaml

```diff
@@ -1,37 +1,29 @@
-version: "3.8"
-
 volumes:
   conda_store_data:
 
 services:
   traefik:
     image: "traefik:v2.6"
-    container_name: "traefik"
     command:
       - "--api.insecure=true"
       - "--providers.docker=true"
-      - "--providers.docker.exposedbydefault=false"
+      - "--providers.docker.exposedbydefault=true"
       - "--entrypoints.web.address=:80"
-      - "--entrypoints.websecure.address=:443"
       - "--entrypoints.minio.address=:9080"
-      - "--entrypoints.websecure.http.tls=true"
-      - "--entrypoints.minio.http.tls=true"
-      - "--entrypoints.web.http.redirections.entryPoint.to=websecure"
-      - "--entrypoints.web.http.redirections.entryPoint.scheme=https"
-      - "--entrypoints.web.http.redirections.entrypoint.permanent=true"
+      - "--entrypoints.websecure.http.tls=false"
+      - "--entrypoints.minio.http.tls=false"
+      - "--entrypoints.web.http.redirections.entryPoint.scheme=http"
     ports:
       - "80:80"
       - "443:443"
-      - "8080:8080"
       - "9080:9080"
     volumes:
       - "/var/run/docker.sock:/var/run/docker.sock:ro"
 
   conda-store-worker:
     image: quansight/conda-store-server
-    user: 1000:1000
+    user: 1000:100
```

Comment:
Is this intended? If so, any context around this would be helpful.

Reply:
This user needs to match the conda store settings `c.CondaStore.default_gid` and `c.CondaStore.default_uid`. In the [conda_store_config.py ](https://github.com/conda-incubator/conda-store/pull/1004/files?diff=unified&w=0#diff-7833f5147a649457fff55e9e16579d23960205f7d475de30802c436467bec33dR24) for this example, the user gid is 100.